### PR TITLE
#2052 - Position pathway reaction on canvas

### PR DESCRIPTION
**Pull request by nupagadi**
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name does not contain '#'
- [x] base branch (master or release/xx) is correct
- [x] PR is linked with the issue
- [x] task status changed to "Code review"
- [x] code follows product standards
- [x] regression tests updated

### CODE REVIEW

File: core/indigo-core/reaction/pathway_reaction.h [hunk 1]
```diff
@@ -33,8 +33,9 @@ namespace indigo
 
     class DLLEXPORT PathwayReaction : public BaseReaction
     {
-        static constexpr int SPACE = 5;
-        static constexpr float NARROWING_QUOTIENT = 0.8f;
+        static constexpr float SPACE = 2.f;
```
Comment (AlexanderSavelyev) [inline core/indigo-core/reaction/pathway_reaction.h:36]: change to MARGIN = 1

**Review by AlexanderSavelyev — CHANGES_REQUESTED** [review]

File: api/tests/integration/tests/reaction/pathway.py [hunk 2]
```diff
@@ -18,7 +18,15 @@ def find_diff(a, b):
 input_path = joinPathPy("reactions/", __file__)
 ref_path = joinPathPy("ref/", __file__)
```
Comment (AliaksandrDziarkach) [inline api/tests/integration/tests/reaction/pathway.py:20]: Could you please add "indigo.setOption("json-saving-pretty", True)" to make ket files readable?

Comment (nupagadi) [reply api/tests/integration/tests/reaction/pathway.py:20]: Thanks for your suggestion. Unfortunately, the files used for this feature are huge, thus beautifying them becomes of a quite questionable benefit. And you don't very often need to see it beautified and online services are at hand in 10 sec.

**Review by AlexanderSavelyev — APPROVED** [review]

File: api/tests/integration/tests/reaction/pathway.py (hunk 2 above)
Comment (AliaksandrDziarkach) [reply api/tests/integration/tests/reaction/pathway.py:20]: It will help when these files changed by other PR

**Pull request merged**